samples/cellular/modem_shell/README.rst

```diff
@@ -1420,7 +1420,7 @@ For example:
 .. parsed-literal::
    :class: highlight
 
-   west build -p -b *board_target* -S nrf91-modem-trace-ext-flash
+   west build -p -b *board_target* -- -DMOSH_SNIPPET="nrf91-modem-trace-ext-flash"
```

Comment:
Shouldn't this be `modem_shell` instead of `MOSH` or that's what we've used now with sysbuild?

```suggestion
   west build -p -b *board_target* -- -Dmodem_shell_SNIPPET="nrf91-modem-trace-ext-flash"
```